NEWS.md

```diff
@@ -14,12 +14,21 @@ expressions (#241)
 ## Breaking Changes
 
 - `assert_order_vars()` was deprecated in favor of `assert_expr_list()`. (#241)
+- The following functions have been deprecated from previous admiral versions using the next phase of the deprecation process: (#272)
+
+  - `quo_c()`
+  - `quo_not_missing()`
+  - `replace_values_by_names()`
+  - `replace_symbol_in_quo()`
+
```

Comment:
were some of these removed completely - can we list them out separately from those that are producing warnings please - jut so we can keep track

Reply:
Removal would be the next step, these were the ones in phase 1 with `deprecate_warn` and I've swapped in the `deprecate_stop`

Reply:
okay cool thank you for checking.

Reply:
can you make a new issue for Q3 release (you can even make the new label! ) that says these four functions should be removed for Q3 release.  Just so we don't lose track

Reply:
`replace_values_by_names()` was not deprecated but the `quosures` argument was replaced by the `expressions` argument.